index.html

```diff
@@ -10,39 +10,250 @@
+        <img src="./image/menu.png" alt="menu bar image" />
+      </nav>
     </header>
     <section class="headline">
-      <h1 class="Primay-text">I'm Abenezer.<br> Glad to see <br>you!</h1>
-      <h3 class="secondary-text">
+      <h1 class="Primay-text">
+        I'm Abenezer.<br />
+        Glad to see <br />you!
+      </h1>
+      <p class="secondary-text">
         I’m a software developer! I can help you build a product , feature or
         website Look through some of my work and experience! If you like what
         you see and have a project you need coded, don’t hesitate to contact me.
-      </h3>
+      </p>
       <div class="social">
         <h3 class="connect">LET'S CONNECT</h3>
         <ul class="social-links">
           <li>
-            <a href="https://twitter.com/" target="_blank"  rel="noopener noreferrer" ><img src="./image/twitter.png" alt="twitter.icon" /></a>
+            <a
+              href="https://twitter.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="./image/twitter.png" alt="twitter.icon"
+            /></a>
           </li>
-          <li><a href="https://www.linkedin.com/in/abenezer-samuel/" target="_blank" rel="noopener noreferrer" ><img src="./image/linkdin.png" alt="linkdin logo"></li></a>
-          
           <li>
-            <a href="" target="_blank" rel="noopener noreferrer" ><img src="./image/medium.png" alt="medium logo" /></a>
+            <a
+              href="https://www.linkedin.com/in/abenezer-samuel/"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="./image/linkdin.png" alt="linkdin logo"
+            /></a>
           </li>
+
           <li>
-           <a href="https://github.com/abenisamuel11" target="_blank"  rel="noopener noreferrer" ><img src="./image/Github.png" alt="github logo" /></a>
+            <a href="" target="_blank" rel="noopener noreferrer"
+              ><img src="./image/medium.png" alt="medium logo"
+            /></a>
           </li>
-          
           <li>
-            <a href="" target="_blank" rel="noopener noreferrer" ><img src="./image/A.png" alt="angle list icon" /></a>
+            <a
+              href="https://github.com/abenisamuel11"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="./image/Github.png" alt="github logo"
+            /></a>
+          </li>
+
+          <li>
+            <a href="" target="_blank" rel="noopener noreferrer"
+              ><img src="./image/A.png" alt="angle list icon"
+            /></a>
           </li>
         </ul>
       </div>
       <br />
     </section>
-    </section>
+    <section class="project-container">
+      <section class="work">
+        <img
+           width="350px"
+          src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl_4x.png?compress=1&resize=400x300&vertical=top"
+        />
```

Comment:
- [ ] I have noticed that the image is not having specified width and therefore it overcrosses the borders of `work` card. I suggest to ensure that the width of the image matches the one of Figma template design with also the border-radius. This is to ensure that the portifolio is user friendly in a such a way that there will be no horizontal scroll. Remember, the experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling. Therefore, I suggest to fix the horizontal scroll also ☺️.
Your design 😎 
![scrool](https://user-images.githubusercontent.com/99394374/236158652-3514dc0c-01cc-47e6-ba70-5d4e38ce2f6e.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/236158723-1187b6e5-aae7-4116-bd32-e109a4ebdc6f.png)